docs/platforms/gitlab-com.md

```
@@ -67,4 +67,4 @@ GitLab provides an option to generate a personal access token for a new user. Fo
 9. Click "Create Personal Access Token."
 10. Note down the token as it will only be displayed once.
 
-![Untitled](/img/integrations/admin-access-token.png)
+![GitLab personal access token configuration page showing token name input, expiration date selector, and API permission scopes including read/write access options](/img/integrations/gitlab-personal-access-token-scopes-setup.png)
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)